scripts/cc-cli/src/commands/send.ts

```diff
@@ -29,8 +35,27 @@ async function sendAction(options: OptionValues) {
 
   // Check options
   checkAmount(options);
-  checkAddress(options.to, api);
+  checkAddress(options.to, api, "send funds to");
+
+  if (options.useEcdsa) {
+    const hash = await sendFromECDSA(options, api);
+    console.log("Transfer transaction hash: " + hash.toHex());
+  } else {
+    const hash = await sendFromSr25519(options, api);
+    console.log("Transfer transaction hash: " + hash.toHex());
```

Comment:
Can we pull the `console.log` call outside the if statement and avoid duplication ?